### PR TITLE
docs(*): renaming classes for 9.0 ivy changes

### PR DESCRIPTION
**Pull request by ChronosSF**

### CODE REVIEW

File: en/components/calendar.md [hunk 1]
```diff
@@ -74,11 +74,11 @@ Notice that the calendar header is not rendered when the selection is either `mu
 
 #### Localization and formatting
 
-Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews).
+Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbasedirective.html#formatviews).
```
Comment (damyanpetev) [inline en/components/calendar.md]: Hold on, why are those links going to the base classes?!
_(Technically should be igxcalendarbase**directive** now, but then again that's not exported and has no place in the API and I marked it hidden.)_

All those links **should** go to the actual component people will interact with:
```suggestion
Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatviews).
```

Comment (ChronosSF) [reply en/components/calendar.md]: Because they were going there previously :)

But I don't mind applying your suggestion ofc, it makes a lot of sense

Comment (ChronosSF) [reply en/components/calendar.md]: They seem to work now though.

https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/igxcalendarbase.html#selection

Comment (ChronosSF) [reply en/components/calendar.md]: I'll rename igxcalendarbase to igxcalendarcomponent but that will probably not be all links pointing to base classes in the docs.

Comment (damyanpetev) [reply en/components/calendar.md]: It makes sense for the grids to use base API in templates for simplicity (though not optimal), but not the single calendar.
And those work now, but like I said I marked it hidden so it should drop out of the API files.

File: en/components/calendar.md [hunk 2]
```diff
@@ -74,11 +74,11 @@ Notice that the calendar header is not rendered when the selection is either `mu
 
 #### Localization and formatting
 
-Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews).
+Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbasedirective.html#formatviews).
```
Comment (damyanpetev) [inline en/components/calendar.md]:
```suggestion
Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatviews).
```

File: en/components/calendar.md [hunk 3]
```diff
@@ -74,11 +74,11 @@ Notice that the calendar header is not rendered when the selection is either `mu
...
 
-In the markup below we are also binding the [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions) and [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews) properties to customize the display formatting. Finally we are binding the [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale) property to a value, based on the user's location choice:
+In the markup below we are also binding the [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions) and [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbasedirective.html#formatviews) properties to customize the display formatting. Finally we are binding the [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale) property to a value, based on the user's location choice:
```
Comment (damyanpetev) [inline en/components/calendar.md]:
```suggestion
In the markup below we are also binding the [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions) and [`formatViews`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatviews) properties to customize the display formatting. Finally we are binding the [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale) property to a value, based on the user's location choice:
```

File: jp/components/calendar.md [hunk 4]
```diff
@@ -76,11 +76,11 @@ import { IgxCalendarComponent } from 'igniteui-angular';
 
 ### ローカライズと書式設定
 
-カレンダーにおいてローカライズや書式設定はとても重要な要素です。[`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) では、その機能が [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale)、[`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions)、および [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews) プロパティによって制御してカスタマイズ化されます。
+カレンダーにおいてローカライズや書式設定はとても重要な要素です。[`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) では、その機能が [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale)、[`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions)、および [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbasedirective.html#formatviews) プロパティによって制御してカスタマイズ化されます。
```
Comment (damyanpetev) [inline jp/components/calendar.md]:
```suggestion
カレンダーにおいてローカライズや書式設定はとても重要な要素です。[`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) では、その機能が [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale)、[`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions)、および [`formatViews`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatviews) プロパティによって制御してカスタマイズ化されます。
```

File: jp/components/calendar.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ import { IgxCalendarComponent } from 'igniteui-angular';
...
+これらの設定に加え、[`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) API のその他のカスタマイズ機能も使用できます。EFTA (欧州自由貿易連合) 国内からページへアクセスする場合、カレンダーを対応するカルチャで表示する必要があります。最初に週の開始日を制御する [`weekstart`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#weekstart) を設定し、デフォルト値が 0 (日曜日) であるため、値を 1 に設定します。
 
-以下のマークアップは、[`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions) および [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews) プロパティをバインドして表示書式設定をカスタマイズします。最後に、[`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale) プロパティをユーザーの選択した場所に基づいて値にバインドします。
+以下のマークアップは、[`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions) および [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbasedirective.html#formatviews) プロパティをバインドして表示書式設定をカスタマイズします。最後に、[`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale) プロパティをユーザーの選択した場所に基づいて値にバインドします。
```
Comment (damyanpetev) [inline jp/components/calendar.md]:
```suggestion
以下のマークアップは、[`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions) および [`formatViews`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatviews) プロパティをバインドして表示書式設定をカスタマイズします。最後に、[`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale) プロパティをユーザーの選択した場所に基づいて値にバインドします。
```

File: jp/components/month_picker.md [hunk 6]
```diff
@@ -61,7 +61,7 @@ import { IgxMonthPickerComponent } from "igniteui-angular";
 ```
 
 #### 日付の設定
-[`IgxMonthPickerComponent`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html) に日付を設定するには、[`value`]({environment:angularApiUrl}/classes/igxcalendarbase.html#value) 入力を設定します。
+[`IgxMonthPickerComponent`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html) に日付を設定するには、[`value`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#value) 入力を設定します。
```
Comment (damyanpetev) [inline jp/components/month_picker.md]:
```suggestion
[`IgxMonthPickerComponent`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html) に日付を設定するには、[`value`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html#value) 入力を設定します。
```

File: jp/components/month_picker.md [hunk 7]
```diff
@@ -84,7 +84,7 @@ Two-way データ バインディングを作成するには、以下のよう
 ```
 
 #### 書式設定
-`formatViews` と [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions) 入力を使用してマンスピッカーの表示形式を変更します。
+`formatViews` と [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions) 入力を使用してマンスピッカーの表示形式を変更します。
```
Comment (damyanpetev) [inline jp/components/month_picker.md]:
```suggestion
`formatViews` と [`formatOptions`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html#formatoptions) 入力を使用してマンスピッカーの表示形式を変更します。
```

File: jp/components/month_picker.md [hunk 8]
```diff
@@ -111,7 +111,7 @@ public numericFormatOptions = {
 </div>
 
 #### ローカライズ
-[`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale) 入力を使用してマンスピッカーのローカライズをカスタマイズします。
+[`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale) 入力を使用してマンスピッカーのローカライズをカスタマイズします。
```
Comment (damyanpetev) [inline jp/components/month_picker.md]:
```suggestion
[`locale`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html#locale) 入力を使用してマンスピッカーのローカライズをカスタマイズします。
```

File: kr/components/month_picker.md [hunk 9]
```diff
@@ -57,7 +57,7 @@ To add a month picker in a template, use the following code:
 ```
 
 #### Setting date
-Set a date to [`IgxMonthPickerComponent`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html) using the [`value`]({environment:angularApiUrl}/classes/igxcalendarbase.html#value) input.
+Set a date to [`IgxMonthPickerComponent`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html) using the [`value`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#value) input.
```
Comment (damyanpetev) [inline kr/components/month_picker.md]:
```suggestion
Set a date to [`IgxMonthPickerComponent`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html) using the [`value`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html#value) input.
```

File: kr/components/month_picker.md [hunk 10]
```diff
@@ -80,7 +80,7 @@ To create a two-way data-binding, set `ngModel` like this:
 ```
 
 #### Formatting
-Change the month picker display format, using the [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions) inputs.
+Change the month picker display format, using the [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions) inputs.
```
Comment (damyanpetev) [inline kr/components/month_picker.md]:
```suggestion
Change the month picker display format, using the [`formatOptions`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html#formatoptions) inputs.
```

File: kr/components/month_picker.md [hunk 11]
```diff
@@ -107,7 +107,7 @@ Here is an example of modifying the default format options of the month picker:
 </div>
 
 #### Localization
-Use the [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale) input, to customize the month picker localization.
+Use the [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale) input, to customize the month picker localization.
```
Comment (damyanpetev) [inline kr/components/month_picker.md]:
```suggestion
Use the [`locale`]({environment:angularApiUrl}/classes/igxmonthpickercomponent.html#locale) input, to customize the month picker localization.
```

**Pull request merged**